### PR TITLE
Adding documentation for @Ignore and dry run

### PR DESCRIPTION
**Pull request by krmahadevan**

### CODE REVIEW

File: doc/documentation-main.html [hunk 1]
```diff
@@ -1622,8 +1622,60 @@ <h3 class="sourcetitle">Test1.java</h3>
 will make both <tt>test1()</tt> and <tt>test2()</tt> test methods but on top of that, <tt>test2()</tt> now belongs to the group "g1".
 <p>
 
+<!-------------------------------------
+  IGNORE TESTS
+  ------------------------------------>
+
+<h4><a class="section" indent=".." name="ignore-tests">Ignoring tests</a></h4>
 
+TestNG now lets you ignore tests in a new way. You can ignore all the <tt>@Test</tt> methods :
```
Comment (cbeust) [inline doc/documentation-main.html]: > TestNG lets you ignore all the <tt>@Test</tt> methods:

(no need for the first sentence and for "now" in a documentation)

File: doc/documentation-main.html [hunk 2]
```diff
@@ -1622,8 +1622,60 @@ <h3 class="sourcetitle">Test1.java</h3>
...
+<ul>
+  <li>In a class (or)</li>
+  <li>In a particular package (or)</li>
+  <li>In a package and all of its child packages</li>
+</ul>
+
+using the new annotation <tt>@Ignore</tt>.
+
+<p>
+The <tt>@Ignore</tt>  when used at the method level, is functionally equivalent to <tt>@Test(enabled=false)</tt>.
```
Comment (cbeust) [inline doc/documentation-main.html]: > "When used at the method level, the <tt>@Ignore</tt> annotation is functionally..."

File: doc/documentation-main.html [hunk 3]
```diff
@@ -1622,8 +1622,60 @@ <h3 class="sourcetitle">Test1.java</h3>
...
+    public void testMethod1() {
+    }
+
+    @Test
+    public void testMethod2() {
+    }
+}
+</pre>
+
+The <tt>@Ignore</tt> annotation has a higher priority. So if use this annotation at the class level, then it disables all tests in the class even though they have a <tt>@Test(enabled=true)</tt>
```
Comment (cbeust) [inline doc/documentation-main.html]: > "The <tt>@Ignore</tt> annotation has a higher priority than individual <tt>@Test</tt> method annotations: when <tt>@Ignore</tt> is placed on a class, all the tests in that class will be disabled."

File: doc/documentation-main.html [hunk 4]
```diff
@@ -2923,6 +2975,18 @@ <h3><a class="section" name="yaml">YAML</a></h3>
 
 You might find the YAML file format easier to read and to maintain. YAML files are also recognized by the TestNG Eclipse plug-in. You can find more information about YAML and TestNG in this <a href="http://beust.com/weblog/2010/08/15/yaml-the-forgotten-victim-of-the-format-wars/">blog post</a>.
 
+<!------------------------------------
+  Dry Run for TestNG
+  ------------------------------------>
+
+<h3><a class="section" name="dryrun">Dry Run for your tests</a></h3>
+
+TestNG now lets you do a dry run of your tests. A dry run is essentially a mode of execution, where TestNG would only tell you what all test methods would get executed, but not actually execute them.
```
Comment (cbeust) [inline doc/documentation-main.html]: "When launched in dry run mode, TestNG will display a list of the test methods that would be invoked but without actually calling them."

File: doc/documentation-main.html [hunk 5]
```diff
@@ -2923,6 +2975,18 @@ <h3><a class="section" name="yaml">YAML</a></h3>
...
+That way you get to cross check if the groups you are running includes all methods that you were expecting or check if the package selection you did was correct etc.,
```
Comment (cbeust) [inline doc/documentation-main.html]: Remove this sentence.

File: doc/documentation-main.html (hunk 1 above)
Comment (krmahadevan) [reply doc/documentation-main.html]: fixed.

File: doc/documentation-main.html (hunk 2 above)
Comment (krmahadevan) [reply doc/documentation-main.html]: fixed

File: doc/documentation-main.html (hunk 3 above)
Comment (krmahadevan) [reply doc/documentation-main.html]: fixed

File: doc/documentation-main.html (hunk 4 above)
Comment (krmahadevan) [reply doc/documentation-main.html]: fixed.

File: doc/documentation-main.html (hunk 5 above)
Comment (krmahadevan) [reply doc/documentation-main.html]: removed

**Pull request merged**